src/environments/environment.prod.ts

```diff
@@ -22,7 +22,7 @@ export const environment: Environment = {
 
   theme: 'default',
   featureFlags: {
-    hideTaskTabs: [ 'hints' ],
+    hideTaskTabs: [],
```

Comment:
This change will be removed before merging

Reply:
The change does not have to be removed since the "hints" tab was disabled just because it was not working.  If it is fixed, it has to be enabled.